### ISSUE TITLE
Changing the file structure

### ISSUE THREAD

**Issue by laytong** [7Geese/eslint-config-7geese#13]
@himynameisdave I had a question about the `on.js`/`off.js`
I can't see anything that describes why it's structured this way and It doesn't follow that structure in the airbnb config. I was wondering if there was a reason or we could squash each on/off pair to a single file?

**Comment by laytong** [comment]
The drive behind this is I'm making a PR to upgrade a lot of the dependencies and there's a lot of new rules we need to cover :)

**Comment by himynameisdave** [comment]
**TLDR;** It was designed this way to make it easier to integrate into non-7Geese projects and be used by the wider community. 

Although it's not done by `airbnb`, it's a pattern that other configs ([Walmart](https://github.com/walmartlabs/eslint-config-walmart/tree/master/rules) & [Formidable](https://github.com/FormidableLabs/eslint-config-formidable/tree/master/rules)) have followed which allows you to compose different rulesets together, based on which ones the consumer actually cares about. [Here's a good overview of that](https://github.com/FormidableLabs/eslint-config-formidable#piecemeal-configurations), but here's the gist:

Let's say you start a new project (at 7Geese or otherwise) which does not use React. You'll want an easy way to turn off those rules:

```json
{
  "extends": [
    "7geese",
    "7geese/rules/jsx-a11y/off",
    "7geese/rules/react/off",
    "7geese/rules/react-hooks/off",
  ]
}
```

☝️ That's the key idea behind composeable configs.

**Comment by laytong** [comment]
riiiight, That explains why you need to duplicate every rule.
Thanks DMoney!

Hopefully a new PR coming soon

**Comment by laytong** [comment]
#14
